lms/djangoapps/instructor/views/api.py

```diff
@@ -3015,6 +3019,10 @@ def remove_certificate_exception(course_key, student):
         )
     except ObjectDoesNotExist:
         # Certificate has not been generated yet, so just remove the certificate exception from white list
+        log.info(u'%s has been removed from the whitelist in course %s',
```

Comment:
I'm not convinced this is a right place for adding this logging. 
You are adding the logging `except` when certificate object is missing.